data/helpers.d/nodejs

```diff
@@ -126,6 +126,9 @@ ynh_install_nodejs () {
 
     # If n is not previously setup, install it
     if ! test $(n --version > /dev/null 2>&1)
+    then
+        ynh_install_n
+    elif dpkg --compare-version "$($n_install_dir/bin/n --version | cut -d" " -f2)" lt "6.5.1"
```

Comment:
To avoid any problems, I would create a new `n_version` variable at the top of this file and use it where needed.